package.json

```diff
@@ -20,13 +20,13 @@
     "react-native": "*"
   },
   "devDependencies": {
-    "@adalo/cli": "^0.0.55",
-    "react": "^16.5.2",
-    "react-art": "^16.6.0",
-    "react-dom": "^16.5.2",
+    "@adalo/cli": "^0.0.57",
+    "react": "^16.13.1",
+    "react-art": "^16.13.1",
+    "react-dom": "^16.13.1",
```

Comment:
Same suggestion as for the other PR, let's block this versions if we can. 

This feedback doesn't block merging the PR though, as it's not a regression (packages weren't locked beforehand)